mpa/modules/models/heads/custom_hierarchical_linear_cls_head.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import normal_init
+
+from mmcls.models.builder import HEADS, build_loss
+from mmcls.models.heads import MultiLabelClsHead
+
+
+@HEADS.register_module()
+class CustomHierarchicalLinearClsHead(MultiLabelClsHead):
+    """Custom Linear classification head for multilabel task.
```

Comment:
Isn't it head for "hierarchical task"? not "multilabel task"

Reply:
I'll revise it, thank you.